com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/Messages.java

```diff
@@ -167,6 +167,10 @@
     public static final String CONTROLLER_CLIENT_CONNECT_TIMEOUT_IN_SECONDS = "Controller client connect timeout in seconds: {0}";
     public static final String CONTROLLER_CLIENT_CONNECTION_POOL_SIZE = "Controller client connection pool size: {0}";
     public static final String CONTROLLER_CLIENT_THREAD_POOL_SIZE = "Controller client thread pool size: {0}";
+    public static final String MICROMETER_DYNATRACE_URL = "Dynatrace URL for micrometer metrics: {0}";
+    public static final String MICROMETER_DYNATRACE_TOKEN = "Micrometer dynatrace token is set: {0}";

```

Comment:
Check whether we could use the ones from our binding to the Dynatrace service instance.